src/infer_response.cc

```diff
@@ -162,4 +163,84 @@ InferResponse::Error()
   return error_;
 }
 
+#ifndef TRITON_PB_STUB
+TRITONSERVER_Error*
+InferResponse::Send(
+    TRITONBACKEND_ResponseFactory* response_factory, void* cuda_stream)
+{
+  // [FIXME] Use this code to send responses in non-decoupled mode.
```

Comment:
Do you mean you would like to use `InferResponse::Send` in non-decoupled mode as well?

Reply:
Right exactly.